wemo_insight_block.py

```diff
@@ -42,8 +42,8 @@ def process_signals(self, signals):
             try:
                 self.device.update_insight_params()
                 self._updating = False
-            except AttributeError:
-                # raised when update_insight_params has given up retrying
+            except:
```

Comment:
I realize that a bare except is a no-no and any decent linter would yell at me, but this is exactly what I intend: catch ALL the things and try again.